sru-scripts/manual/oracle-sru

```diff
@@ -32,13 +32,13 @@ done
 # Capture current state
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- dpkg-query --show cloud-init
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- sudo cat /run/cloud-init/result.json
-ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- ! grep Trace /var/log/cloud-init.log
+ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- ! grep Trace /var/log/cloud-init.log || true  # Oracle can give us some tracebacks, so we should check the output manually for tracebacks
```

Comment:
@TheRealFalcon oops I see you added the comment here. Feel free to disregard the other comment. or at least to add detail about OpenStack datasource not fully supporting Oracle.